ramses_rf/devices_hvac.py

```diff
@@ -251,44 +251,102 @@ class HvacVentilator(DeviceHvac):  # FAN: RP/31DA, I/31D[9A]
     _SLUG: str = DEV_TYPE.FAN
 
     @property
-    def boost_timer(self) -> Optional[int]:
-        return self._msg_value(_31DA, key="remaining_time")
+    def exhaust_fan_speed(self) -> Optional[float]:
+        return self._msg_value((_31DA), key=EXHAUST_FAN_SPEED)
+        
```

Comment:
Rename `fan_rate` instead - not it uses the latest value of the two packets - see below: